### DISCUSSION TITLE
Easiest way to update the traccar-server.jar?

### DISCUSSION THREAD

**Discussion by zcsrf** [hassio-addons/addon-traccar#137] [Q&A] [answered]
Hello Everyone :) 

Currently I want to change the way some Traccar protocols are decoded (mainly define some more io's in teltonika) what would be the easiest way to change the traccar-server.jar from the docker?

Thanks

**Comment by frenck** [comment, marked as answer]
That is not supported by add-ons.

**Comment by zcsrf** [comment]
Hi Frank, 

On that case, the best way would be to fork this addon and change the dockerfile to grab my traccar-server.jar correct?

Thanks for the help.

**Reply by frenck** [reply]
That is up to you
